packages/api-clients/package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "lint": "eslint . --ext .ts,.tsx",
+    "lint:openapi": "spectral lint -D ./open-api/!\\(selfcareV2ClientApi.yml\\|authorizationManagementApi.yaml\\|notifierApi.yml\\) --ruleset .spectral.yaml",
```

Comment:
Here you said that you moved the files to exclude into the spectral config: https://github.com/pagopa/interop-be-monorepo/pull/1719#discussion_r2137983786

But I still see them inline both here and in the PR workflow